src/pages/trading-specification/components/_calculationCard.tsx

```diff
@@ -18,13 +18,13 @@ const CalculationCard = ({ data }: DlMarketTypeProps) => {
             </Header>
 
             {data.list.map((list, index) => (
-                <>
-                    <Header key={index} type="paragraph-2" weight="normal" as="p">
+                <React.Fragment key={index}>
+                    <Header type="paragraph-2" weight="normal" as="p">
```

Comment:
```suggestion
                    <Header key={index} type="paragraph-2" weight="normal" as="p">
```

Reply:
`<Header>` and `<img>` elements are inside `.map` and it require one parent element.